src/cgame/cg_rocket_draw.cpp

```diff
@@ -3526,46 +3531,50 @@ static void CG_Rocket_HaveJetpck()
 struct elementRenderCmd_t
 {
 	const char *name;
-	void ( *exec )();
+	void ( *update )(); // Modifies the RML or element properties
+	void ( *render )(); // Does rendering outside of the HTML engine
 	rocketElementType_t type;
 };
 
+// This kind of element is a hack left over from when VMs could only use C code
```

Comment:
just asking, maybe this could use the `deprecate` thingy of C++14? Or a TODO, since this looks like a pretty good candidate of things to get rid of soon (those manually sorted list are quite annoying, but at least here it's documented...)

Reply:
True, but it might help encourage us to fix the situation if we see them :)